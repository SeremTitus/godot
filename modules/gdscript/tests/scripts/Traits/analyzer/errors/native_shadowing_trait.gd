extends CharacterBody2D
uses BadTrait

trait BadTrait:
    extends CollisionObject2D
    func move_and_collide()
