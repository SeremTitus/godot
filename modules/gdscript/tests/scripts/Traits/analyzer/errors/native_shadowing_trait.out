GDTEST_ANALYZER_ERROR
>> ERROR at line 2: "move_and_collide" function from trait "BadTrait" shadows a native method of the using class.
